common-lib/src/main/scala/com/gu/mediaservice/lib/logging/MarkerUtils.scala

```diff
@@ -2,15 +2,28 @@ package com.gu.mediaservice.lib.logging
 
 import net.logstash.logback.marker.LogstashMarker
 import net.logstash.logback.marker.Markers.appendEntries
+import play.api.MarkerContext
+import scala.language.implicitConversions
+
 import scala.collection.JavaConverters._
 
-trait LogMarker { def toLogMarker: LogstashMarker }
+trait LogMarker {
+  def toLogMarker: LogstashMarker
+  def map: Map[String, Any]
```

Comment:
can we name this to be more intention revealing?

Reply:
it's now called `markerContents`